### DISCUSSION TITLE
TP3 - Duda ejercicio 1

### DISCUSSION THREAD

**Discussion by NicoHermosilla** [INGCOM-UNRN-P1/cursada-2023#103] [Preguntas y respuestas] [answered]
Consulta en el pto 1.1, queria saber si esta mal definir la capacidad del arreglo estatica y despues modificarlo con lo ingresado por el usuario o directamente tomo lo definido por el usuario
https://github.com/INGCOM-UNRN-P1/tp3-2023-NicoHermosilla

**Comment by martinvilu** [comment, marked as answer]
La capacidad, tiene que ser parte de los argumentos de las funciones, indistintamente de la capacidad real de los arreglos que, si, van a ser estáticos. La cosa es que el código nunca lo sabrá.
